pool/position_modify.gno

```diff
@@ -10,16 +10,16 @@ import (
 
 // modifyPosition updates a position in the pool and calculates the amount of tokens to be added or removed.
 // Returns positionInfo, amount0, amount1
-func (pool *Pool) modifyPosition(params ModifyPositionParams) (PositionInfo, *i256.Int, *i256.Int) {
+func (pool *Pool) modifyPosition(params ModifyPositionParams) (PositionInfo, *u256.Uint, *u256.Uint) {
```

Comment:
Changing return types needs clear reason. Please give us explanation

Reply:
In the previous implementation, the `amount0` and `amount1` values obtained from the this function were always being converted using `Abs` method.

Therefore, to reduce unnecessary process, I modified the function to apply `Abs` internally before returning the values, which resulted in changing the return type to `*i256.Int`.